app/src/main/java/com/jerboa/ui/components/post/PostListings.kt

```diff
@@ -57,7 +57,8 @@ fun PostListings(
 ) {
     SwipeRefresh(
         state = rememberSwipeRefreshState(loading),
-        onRefresh = onSwipeRefresh
+        onRefresh = onSwipeRefresh,
+        indicatorPadding = padding
     ) {
```

Comment:
We've got to get rid of these at some point anyway, because swiperefresh is deprecated.